### ISSUE TITLE
Would a mirror be helpful?

### ISSUE THREAD

**Issue by C-Otto** [coinbase/toshi#224]
I run a mirror server with lots of storage and network connectivity. Would it be of any real value to add another public toshi instance on it?

**Comment by phuongnd08** [comment]
Yes, it is. Please do :)

**Comment by C-Otto** [comment]
I'll consider it. How would it be used? Is there some page that advertises our server (mirror)? Is there actual demand?

**Comment by phuongnd08** [comment]
I think many start up who want to access to Bitcoin would use this. bitcoin.toshi.io can be down from time to time and a spare node of it on the internet would be helpful. I my self really think it's a waste to dup a node like bitcoin.toshi.io over and over since a lot of storage is demanded (right now around 1TB).

**Comment by C-Otto** [comment]
OK. I guess you're in charge of `bitcoin.toshi.io`? Would you be willing to announce and link to our mirror when it is there? I'm asking as I don't want to invest so much time (and disk space etc.) without any user knowing about it, and I also don't think that I'd do a good job at advertising the mirror.

**Comment by phuongnd08** [comment]
Nope, I'm not. @aalness Are you in charge of bitcoin.toshi.io?

**Comment by benzmuircroft** [comment]
I would be interested!

**Comment by aalness** [comment]
@phuongnd08 nope. I no longer work at Coinbase. My guess is it's probably down due to hitting the 1TB limit on the RDS instance it was using. I suspect vacuuming would likely free up a lot of space. Other than that it will need to migrate to a bigger RDS instance.
